float32 theta_0
int16 bicep_volt
int16 tricep_volt
bool cal_bool